usr/initiator.c

```diff
@@ -2142,11 +2145,13 @@ int session_logout_task(int sid, queue_task_t *qtask)
 		/* LLDs that offload login also offload logout */
 		if (!(session->t->caps & CAP_LOGIN_OFFLOAD)) {
 			/* unbind is not supported so just do old logout */
-			if (!iscsi_send_logout(conn))
+			rc = iscsi_send_logout(conn);
+			if (rc)
 				return ISCSI_SUCCESS;
 		}
 
-		conn_error(conn, "Could not send logout pdu. Dropping session");
+		conn_error(conn, "Could not send logout pdu(%s) from session_logout_task."
+			"Dropping session", strerror(rc));
```

Comment:
Nit: you have no space between your two text strings. In general, please do NOT break messages like this over two lines, as that makes it hard to search for (and read) messages.